### PR TITLE
Updated pingsource docs, collated info

### PR DESCRIPTION
**Pull request by abrennan89**
First part of the fix for https://github.com/knative/docs/issues/3507

## Proposed Changes

- Collate PingSource docs into one place
- Editing / review of content

### CODE REVIEW

File: docs/eventing/sources/ping-source/index.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
+
+A PingSource is an event source that produces events with a fixed payload on a specified [cron](https://en.wikipedia.org/wiki/Cron) schedule.
+
+The following example shows how you can configure a PingSource as an event source that sends events every minute to a Knative service named `event-display` that is used as a sink.
+
+## Before you begin
+
+1. To create a Knative service to use as a sink, you must install [Knative Serving](../../../install).
```
Comment (lionelvillard) [inline docs/eventing/sources/ping-source/index.md:19]: Personally I would prefer to not have to install Serving just to try this example.

File: docs/eventing/sources/ping-source/index.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/1 * * * *"
+  contentType: "application/json"
+  data: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: v1
```
Comment (lionelvillard) [inline docs/eventing/sources/ping-source/index.md:81]: If you use Knative Serving, it should be `serving.knative.dev/v1`

File: docs/eventing/sources/ping-source/index.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    kind: PingSource
+    metadata:
+      name: test-ping-source-binary
+    spec:
+      schedule: "*/1 * * * *"
+      contentType: "text/plain"
+      dataBase64: "ZGF0YQ=="
+      sink:
+        ref:
+          apiVersion: v1
```
Comment (lionelvillard) [inline docs/eventing/sources/ping-source/index.md:120]: same here

File: docs/eventing/sources/ping-source/index.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+```shell
+kn source ping delete test-ping-source-binary
+```
+
+    {{< /tab >}}
+    {{< /tabs >}}
+    <br>
+
+
+1. Optional: Delete the `event-delivery` service:
```
Comment (lionelvillard) [inline docs/eventing/sources/ping-source/index.md:223]: nit: `event-delivery` -> `event-display`

File: docs/eventing/sources/ping-source/index.md (hunk 1 above)
Comment (abrennan89) [reply docs/eventing/sources/ping-source/index.md:19]: Is that possible though? I.e. how would someone use or test a PingSource is working w/o a service?

Comment (lionelvillard) [reply docs/eventing/sources/ping-source/index.md:19]: The current doc shows how to use a k8s service: https://knative.dev/docs/eventing/sources/pingsource/#creating-the-event-display-service

File: docs/eventing/sources/ping-source/index.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+title: "PingSource"
+weight: 31
+type: "docs"
+aliases:
+  - /docs/eventing/sources/pingsource
+  - ../cronjob-source
+  - /docs/eventing/samples/ping-source
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
```
Comment (lionelvillard) [inline docs/eventing/sources/ping-source/index.md:11]: I originally intended to assign an unique color based on the maturity of the API: alpha = red, beta = orange, v1 = green. WDYT?

Comment (abrennan89) [reply docs/eventing/sources/ping-source/index.md:11]: I'm OK with that however I think it should be tracked somewhere which docs have which labels to make sure they're kept up to date, in general but especially if we're color coding them.

Comment (abrennan89) [reply docs/eventing/sources/ping-source/index.md:11]: @lionelvillard should this one be orange or green? I'm not clear on whether we want orange for beta or green for v1 for something that's called `v1beta2`

File: docs/eventing/sources/ping-source/index.md (hunk 2 above)
Comment (abrennan89) [reply docs/eventing/sources/ping-source/index.md:81]: Removed mentions of serving so not sure if this is still relevant?

File: docs/eventing/sources/ping-source/index.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    <br>
+
+1. Create a PingSource that sends an event containing `{"message": "Hello world!"}` every minute, by entering the command:
+
+    {{< tabs name="create-source" default="kn" >}}
+    {{% tab name="YAML" %}}
+
+```shell
+kubectl create -n pingsource-example -f - <<EOF
+apiVersion: sources.knative.dev/v1beta2
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/index.md:83]: v1beta2 -> v1

File: docs/eventing/sources/ping-source/index.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+
+1. Optional: Create a PingSource that sends binary data.
+
+    If you want to send binary data in an event, this cannot be directly serialized in YAML. However, you can use `dataBase64` in place of `data` in the PingSource spec to carry a data payload that is base64 encoded.
+
+    To create a PingSource that uses base64 encoded data, enter the command:
+
+    ```shell
+    kubectl -n pingsource-example apply -f - <<EOF
+    apiVersion: sources.knative.dev/v1beta2
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/index.md:122]: v1beta2 -> v1

File: docs/eventing/sources/ping-source/index.md (hunk 5 above)
Comment (abrennan89) [reply docs/eventing/sources/ping-source/index.md:11]: Update to v1

File: docs/eventing/sources/ping-source/index.md [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+title: "PingSource"
+weight: 31
+type: "docs"
+aliases:
+  - /docs/eventing/sources/pingsource
+  - ../cronjob-source
+  - /docs/eventing/samples/ping-source
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/index.md:11]:
```suggestion
![version](https://img.shields.io/badge/API_Version-v1-green?style=flat-square)
```

File: docs/eventing/sources/ping-source/index.md [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+    <br>
+
+1. Create a PingSource that sends an event containing `{"message": "Hello world!"}` every minute, by entering the command:
+
+    {{< tabs name="create-source" default="kn" >}}
+    {{% tab name="YAML" %}}
+
+```shell
+kubectl create -n pingsource-example -f - <<EOF
+apiVersion: sources.knative.dev/v1beta2
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/index.md:83]:
```suggestion
apiVersion: sources.knative.dev/v1
```

File: docs/eventing/sources/ping-source/index.md [hunk 10]
```diff
@@ -0,0 +1,252 @@
...
+
+1. Optional: Create a PingSource that sends binary data.
+
+    If you want to send binary data in an event, this cannot be directly serialized in YAML. However, you can use `dataBase64` in place of `data` in the PingSource spec to carry a data payload that is base64 encoded.
+
+    To create a PingSource that uses base64 encoded data, enter the command:
+
+    ```shell
+    kubectl -n pingsource-example apply -f - <<EOF
+    apiVersion: sources.knative.dev/v1beta2
```
Comment (abrennan89) [inline docs/eventing/sources/ping-source/index.md:122]:
```suggestion
    apiVersion: sources.knative.dev/v1
```

**Pull request merged**